app/controllers/concerns/user_tracking_concern.rb

```diff
@@ -27,4 +27,9 @@ def user_needs_sign_in_update?
   def user_needs_feed_update?
     current_user.last_sign_in_at < REGENERATE_FEED_DAYS.days.ago
   end
+
+  def regenerate_feed!
+    Redis.current.setnx("account:#{current_user.account_id}:regeneration", true) == 1 && Redis.current.expire("account:#{current_user.account_id}:regeneration", 3_600 * 24)
```

Comment:
SETNX + EXPIRE can be performed by one redis command. https://redis.io/commands/set
Redis-rb seems support them. https://github.com/redis/redis-rb/blob/master/lib/redis.rb#L735